test/integration/header_integration_test.cc

```diff
@@ -42,7 +42,11 @@ const std::string http_connection_mgr_config = R"EOF(
   - name: envoy.filters.http.router
 codec_type: HTTP1
 use_remote_address: false
-xff_num_trusted_hops: 1
+original_ip_detection_extensions:
```

Comment:
Nice to have this updated.

It occurs to me belatedly we should probably update docs/root/configuration/http/http_conn_man/headers.rst to refer to the extension.

Reply:
ah yes -- that's very important. I'll drop something under the `x-forwarded-for` section.

Reply:
Added something -- but I think it needs more work. I'll do another pass tomorrow.